10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>I would suggest moving these parameters down to the code blocks where they are used. At this point in the tutorial, the user has no idea what the parameters mean or what they are used for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>